src/pages/workspace/accounting/qbd/export/QuickbooksDesktopExportPage.tsx

```diff
@@ -0,0 +1,120 @@
+            title: qbdConfig?.reimbursableExpensesExportDestination ? translate(`workspace.qbo.accounts.${qbdConfig?.reimbursableExpensesExportDestination}`) : undefined, // TODO: should be updated to use new routes
+            subscribedSettings: [CONST.QUICKBOOKS_CONFIG.REIMBURSABLE_EXPENSES_EXPORT_DESTINATION, CONST.QUICKBOOKS_CONFIG.REIMBURSABLE_EXPENSES_ACCOUNT],
+        },
+        {
+            description: translate('workspace.qbo.exportInvoices'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_INVOICE_ACCOUNT_SELECT.getRoute(policyID)), // TODO: should be updated to use new routes
+            title: qbdConfig?.receivableAccount?.name,
+            subscribedSettings: [CONST.QUICKBOOKS_CONFIG.RECEIVABLE_ACCOUNT],
+        },
+        {
+            description: translate('workspace.accounting.exportCompanyCard'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_COMPANY_CARD_EXPENSE_ACCOUNT.getRoute(policyID)), // TODO: should be updated to use new routes
+            brickRoadIndicator: qbdConfig?.errorFields?.exportCompanyCard ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: qbdConfig?.nonReimbursableExpensesExportDestination ? translate(`workspace.qbo.accounts.${qbdConfig?.nonReimbursableExpensesExportDestination}`) : undefined,
+            subscribedSettings: [
+                CONST.QUICKBOOKS_CONFIG.NON_REIMBURSABLE_EXPENSES_EXPORT_DESTINATION,
+                CONST.QUICKBOOKS_CONFIG.NON_REIMBURSABLE_EXPENSE_ACCOUNT,
+                ...(shouldShowVendorMenuItems ? [CONST.QUICKBOOKS_CONFIG.AUTO_CREATE_VENDOR] : []),
+                ...(shouldShowVendorMenuItems && qbdConfig?.autoCreateVendor ? [CONST.QUICKBOOKS_CONFIG.NON_REIMBURSABLE_BILL_DEFAULT_VENDOR] : []),
+            ],
+        },
+        {
+            description: translate('workspace.qbo.exportExpensifyCard'),
+            title: translate('workspace.qbo.accounts.credit_card'),
+            shouldShowRightIcon: false,
+            interactive: false,
+        },
+    ];
+
+    return (
+        <ConnectionLayout
+            displayName={QuickbooksDesktopExportPage.displayName}
+            headerTitle="workspace.accounting.export"
+            title="workspace.qbd.exportDescription"
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN]}
+            policyID={policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            contentContainerStyle={styles.pb2}
+            titleStyle={styles.ph5}
+            shouldBeBlocked={!canUseNewDotQBD} // TODO: remove it once the QBD beta is done
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.QBO} // TODO: should be updated to use the new connection
+            onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING.getRoute(policyID))}
```

Comment:
Looks like having a fallback to the `goBack` function here led to the following issue:
- https://github.com/Expensify/App/issues/48630

which was fixed by removing the fallback, more details in the [proposed solution](https://github.com/Expensify/App/issues/51107#issuecomment-2423370557).

Reply:
Thanks for the heads up @ikevin127.

cc @ZhenjaHorbach @hoangzinh for visibility